README.md

```diff
@@ -206,32 +227,41 @@ When the insert-variable-plugin is enabled, users will have the option to insert
 ## rdfa-date-plugin
```

Comment:
Can you add an example for the date-insert widget?

Reply:
yes sorry, I forgot about that part